core/Tracker/Visit/VisitProperties.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Tracker\Visit;
+
+/**
+ * Holds temporary data for tracking requests.
+ *
+ * RequestProcessors
+ */
+class VisitProperties
+{
+    /**
+     * Information about the current visit. This array holds the column values that will be inserted or updated
+     * in the `log_visit` table, or the values for the last known visit of the current visitor.
+     *
+     * @var array
+     */
+    public $visitorInfo = array();
```

Comment:
Should we provide a method instead? As in `getAllVisitorInfo()` and `getVisitorInfo($columnName)`? Also the `$visitor` part in `$visitorInfo` is maybe a bit duplicated and could be just `$info`. `$info` is a very generic word as `$data` etc. Maybe we can name it differently. `getAttributes()` and `getAttribute($columnName)` is probably not much better either, maybe there is a good, natural wording for it?